### DISCUSSION TITLE
How & where do I create a Linkfree profile ?

### DISCUSSION THREAD

**Discussion by Richa-24-wav** [EddieHubCommunity/support#6469] [First-Timers] [answered]
I'm looking to create my own link-free profile ! How do I create one ?

**Comment by NaNshekhar04** [comment, marked as answer]
Check this out ! Follow the following steps to get started !

https://linkfree.eddiehub.io/docs/quickstart

**Comment by Richa-24-wav** [comment]
@NaNshekhar04  Thanks a ton it really helped !
